package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "wp-job-manager",
 	"title": "WP Job Manager",
-	"version": "1.34.2",
+	"version": "1.34.3",
```

Comment:
`package-lock` is still with `"version": "1.34.2"`.
The dependencies look okay! I ran the `npm i` and it updated only the version.

Reply:
👍  1d0b8898bf21dfc5b47deb3c39974735a9696947